bundle/edu.gemini.model.p1/src/test/scala/edu/gemini/model/p1/dtree/RootSpec.scala

```diff
@@ -12,8 +12,8 @@ class RootSpec extends SpecificationWithJUnit {
       root.choices must contain(Instrument.Gsaoi)
     }
     "include Texes" in {
-      val root = new Root(Semester(2015, A))
-      root.choices must not contain (Instrument.Texes)
+      val root = new Root(Semester(2016, A))
```

Comment:
I wonder if all tests here should/could be changed to 2016A? I guess the aim here is to check that for the upcoming semester all relevant instruments are available (or not available).